app/views/npq_separation/admin/applications/reviews/index.html.erb

```diff
@@ -125,7 +146,7 @@
           row.with_cell(text: boolean_red_green_tag(application.eligible_for_funding))
           row.with_cell(text: lead_provider_approval_status_badge(application.lead_provider_approval_status))
           row.with_cell(text: application.notes)
-          row.with_cell(text: govuk_link_to("View", npq_separation_admin_application_review_path(application.id)))
+          row.with_cell(text: govuk_link_to("View application", npq_separation_admin_application_review_path(application.id)))
```

Comment:
Whilst this does make this clearer, it is providing context the user already has. Given a list of applications, with a link to view - a default expectation would be to view the application that show is showing.

This has the disadvantage that it also takes up significantly more horizontal space - the screen grabs are all using 'short demon' names for both user and school - I think with longer names, which they are likely to be in reality, then this will result in `View application` potentially wrapping.

Whilst

Reply:
Yea I don't have a strong opinion on this - happy to remove 'application'.